LiteCore/Storage/SQLiteDataFile.cc

```diff
@@ -557,6 +557,11 @@ namespace litecore {
 
             keyStore->addExpiration();
             deletedStore->addExpiration();
+
+            // Add index on sequences
+            // Requiement for RevTreeRecord::read()
+            keyStore->createSequenceIndex();
+            deletedStore->createSequenceIndex();
```

Comment:
Do we also need the sequence index for the deleted table? Basically, do we have query by sequence on the deleted table?

Reply:
From the Profiling report, RevTreeRecord::read does call BothKeyStore::read, which proceeds to call read on the live and dead store. Based on the code in RevTreeRecord::read, I think those would both be read with Sequence.